environment.yml

```
@@ -8,6 +8,7 @@ channels:
 dependencies:
   - python=3.8
   - pip
+  - setuptools_scm
```

You need toml as well if you use pyproject.toml.
```suggestion
  - setuptools_scm
  - toml
```